### ISSUE TITLE
Pools not loading

### ISSUE THREAD

**Issue by UniverseSwap** [VitaliyShulik/launchpad#32]
Good day! Faced the problem of loading pools. They are created, but you should go from another device. We see an eternal loading. What could be the problem ? Thank you

**Comment by VitaliyShulik** [comment]
Hello, the app is using WebSocket RPC. Please check this tip from the addNewNetwork doc:
https://github.com/VitaliyShulik/launchpad/blob/2562f7ff95f481e31c8aa39aad133d54390cf8e8/docs/addNewNetwork.md?plain=1#L23-L25

**Comment by UniverseSwap** [comment]
but I have the first block in theory.  Need to set the  first block for creating a smart contract ido factory?
![Error4](https://user-images.githubusercontent.com/128313604/231374928-c04692c3-075e-4593-bd5c-19484d38d769.png)

**Comment by VitaliyShulik** [comment]
> but I have the first block in theory. Need to set the first block for creating a smart contract ido factory? ![Error4](https://user-images.githubusercontent.com/128313604/231374928-c04692c3-075e-4593-bd5c-19484d38d769.png)

Yes, you can use a block when an IDOFactory smart contract was deployed.

**Comment by UniverseSwap** [comment]
https://api.universeswap.trade
https://filfox.info/en/address/0xEADEDaB3Bf182fD05825A6b3203917D35A8d373A
go and see for yourself. We have a constant load

**Comment by UniverseSwap** [comment]
The settings are correct too. We can create a pool, but it doesn't show up afterwards. Log in from another device or press ctrl+f5

**Comment by UniverseSwap** [comment]
maybe we need to increase the download time? Due to the fact that it is slow, it does not have time to load?

**Comment by VitaliyShulik** [comment]
try to use 2760550 from block

**Comment by UniverseSwap** [comment]
Fine. Thank you. I will post the result

**Comment by UniverseSwap** [comment]
it didn't help us. I went to your test site. According to the smart contract, I see that you have 2 launches in your history, but they are not visible. Eternal loading
![testt](https://user-images.githubusercontent.com/128313604/231386336-ec6dc45d-e9af-4ef8-8d1f-199a6e146376.png)

**Comment by UniverseSwap** [comment]
but after a while you got loaded. Blockchain BSC is faster than FVM, so I think we need to increase the request time

**Comment by UniverseSwap** [comment]
attempts to find the problem led us to the same error. We have changed the key storage network to Filecoin.
But still no result.
![Error6](https://user-images.githubusercontent.com/128313604/231390587-cdfb475e-b436-474c-be04-b9ac16ffb7fc.png)

**Comment by UniverseSwap** [comment]
we will transfer big money to your address. Just help solve the problem. Deadlines are burning

**Comment by VitaliyShulik** [comment]
There are no any of my addresses... You could transfer tokens only on your created IDO Pools.

I didn't test it with an FVM-like chain and not sure that it works with this... Try to change a WebSocket RPC URL.

**Comment by UniverseSwap** [comment]
socket problem. I found the reason.

**Comment by UniverseSwap** [comment]
None of the networks worked for me. I went back to the cob namely BSC. I created a node and got my socket. After that I earned BSC. Therefore, I see that the problem is in the sockets

**Comment by VitaliyShulik** [comment]
Okay, good luck!

**Comment by UniverseSwap** [comment]
thank you too. We will make a donation after the launch. We don’t forget developers with snd, we themselves are like that)

**Comment by RostislavStoliarchuk** [comment]
> Hello, the app is using WebSocket RPC. Please check this tip from the addNewNetwork doc:
> 
> https://github.com/VitaliyShulik/launchpad/blob/2562f7ff95f481e31c8aa39aad133d54390cf8e8/docs/addNewNetwork.md?plain=1#L23-L25

Hello! I ran into a problem with Websocket. Getblock works, but ankr, quicknode, alchemy don't. They issue status 200 and 413 on requests. What could be the problem?

**Comment by RostislavStoliarchuk** [comment]
![#6](https://user-images.githubusercontent.com/88757013/233933817-2f285a55-f684-4bf2-a928-73e40dbbcabc.png)
